Testscripts/Linux/dpdkUtils.sh

```diff
@@ -180,7 +180,23 @@ function Install_Dpdk() {
 	if [[ $DPDK_LINK =~ .tar ]]; then
 		ssh ${install_ip} "mkdir ${DPDK_DIR}"
 		dpdk_download_file="${DPDK_LINK##*/}"
-		ssh "${install_ip}" "wget '${DPDK_LINK}' -P /tmp"
+		retries=0
```

Comment:
Can you remove duplicated block?

Reply:
there is an issue with duplicate code removal for DPDK, but I can add this method to utils.sh, as it is a pretty useful wrapper.